tareas/Tarea1-miguelraz.jl

```diff
@@ -0,0 +1,100 @@
+# # Tarea 1
+
+# ---
+#
+# ## 1. Triángulo de Pascal
+#
+# a. Generen el triángulo de Pascal, hasta cierto orden `ord`,
+# usando una matriz de enteros, en la que cada renglón representa un orden
+# distinto. Hagan que la apariencia de este renglón sea lo más simétrico posible.
+#
+# b. Usen la matriz creada para generar *otra*, en que todos los números
+# pares aparezcan como `false` y los impares como `true`, o alternativamente
+# como 0 y 1, respectivamente. Las funciones `isodd` o `iseven` pueden serles útiles.
+#
+# c. Dibujen (con puntos) todos los valores impares del triángulo de Pascal
+# para `ord=256`. Repitan el ejercicio para `ord=1024`.
+#
+# NOTA: La parte `a` y `b` la pueden lograr con una función; si bien esto es cómodo
+# pueden ir paso a paso.
+function pascal(n)
+           mat = zeros(Int, n+1,2n+1)
+           mat[1, n + 1] = 1
+           for i in 2:n+1
+               for j in 2:2n
+                   mat[i, j] = mat[i-1, j - 1] + mat[i - 1, j + 1]
+               end
+           end
+           mat[n+1, 1] = 1
+           mat[n+1, 2n + 1] = 1
+           mat
+       end
+
+respuesta_b(n) = isodd.(pascal(n))
+
+respuesta_b(256)
+using UnicodePlots
+
+spy(respuesta_b(256))
+
+spy(respuesta_b(1024))
+
```

Comment:
Empiezo diciendo que el código es correcto, y que es buena idea el uso de `UnicodePlots` y `spy`. Sin embargo, como está ahora, no es "reproducible" en relación al `Project.toml` y `Manifest.toml` actuales. Hay al menos dos opciones:
(a) Actualiza `Project.toml` y `Manifest.toml` al agregar `UnicodePlots`; o,
(b) Usa `Plots` (también tiene `spy`)
Insisto, cualquiera de las dos opciones es buena.

Reply:
Además, sugiero separar las "celdas" si se genera el ipynb (con `#-`, nota que no hay espacio después de `#`):

```suggestion
respuesta_b(n) = isodd.(pascal(n))

#-
respuesta_b(256)

#-
using UnicodePlots

spy(respuesta_b(256))

#-
spy(respuesta_b(1024))

```